### ISSUE TITLE
debugLogPath variable should be from config

### ISSUE THREAD

**Issue by edvinasnet** [stackify/stackify-api-php#13]
Hi all,

In AbstractTransport.php constructor:
```php
    public function __construct()
    {
        $ds = DIRECTORY_SEPARATOR;
        $this->debugLogPath = realpath(dirname(__FILE__) . "$ds..$ds..") . $ds . 'debug/log.log';
        $this->errorGovernor = new ErrorGovernor();
        // add empty implementation to avoid method calls on non-object
        $this->setMessageBuilder(new NullBuilder());
    }
```

This debugLogPath is hardcoded with constant value in vendor folder. Maybe its good idea to have configurable variable?

**Comment by homiedopie** [comment]
resolved #15

@edvinasnet - Thank you for your time and suggestion. If you are still keen on checking the changes. You may try out the last version **1.3.0**

**Comment by homiedopie** [comment]
Closing for now. Will reopen if issue still persist.
